### DISCUSSION TITLE
useFieldArray example not working

### DISCUSSION THREAD

**Discussion by ghost** [react-hook-form/react-hook-form#5374] [Q&A] [answered]
I wanted to see how to do validation with Yup using useFieldArray and saw in the examples you might have had something, however when going here there is just an error

https://codesandbox.io/s/react-hook-form-fieldsarray-yup-validation-min-length-k87iy?file=/src/index.js:176-184

I tried to modify it a bit on my end by putting Yup in front of object eg: Yup.object like below but then I get another error that says 'array is not defined'

```
const schema = Yup.object().shape({
  items: array()
    .of(
      Yup.object().shape({
        lineItem: string().required("Some text is required"),
      })
    )
    .required(),
});
```

**Comment by vicke4** [comment, marked as answer]
The import statement has to be,

```js
import { yupResolver } from "@hookform/resolvers/yup";
```

instead of 

```js
import { yupResolver } from "@hookform/resolvers";
```

**Comment by ghost** [comment]
Aha, thank you. The example also breaks when I match the versions to my versions I am running.

` <p>Errors: {JSON.stringify(errors)}</p>`

```
TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'HTMLInputElement'
    |     property '__reactFiber$9otc8usmocu' -> object with constructor 'FiberNode'
    --- property 'stateNode' closes the circle

```

**Comment by ghost** [comment]
This error happens when using the following versions of @hookform/resolvers, yup and react hook form

`@hookform/resolvers": "^2.5.1",`
"react-hook-form": "^7.6.1",
"yup": "^0.32.9"
